pandas/core/ops/array_ops.py

```diff
@@ -386,8 +379,13 @@ def get_array_op(op, str_rep: Optional[str] = None):
 
     Returns
     -------
-    function
+    functools.partial
     """
+    if isinstance(op, partial):
+        # We get here via dispatch_to_series in DataFrame case
+        # TODO: avoid getting here
```

Comment:
What is an example with which you get this?

Reply:
In _arith_method_FRAME we pass na_op (which is one a partial returned by this function here) to _combine_frame, which in turn passes na_op to dispatch_to_series, which then passes na_op back to this function